app/db/init_data.py

```diff
@@ -117,14 +118,14 @@ def import_species(self) -> None:
         logger.info("Importing species")
 
         for record_id, sp in self.species.items():
-            logger.info(f"Importing species: {sp['name']} ({record_id})")
+            logger.info(f"Importing species: {sp['input']} ({record_id})")
```

Comment:
This throws `KeyError`. I don't see an `input` attribute on `sp`. Have you changed something with your data?

Reply:
Same issue with lines 128 and 145.

Reply:
So the data folder I had was the latest one with the `input` and `inputId` fields. I thought that the newest version of Data has new fields and we have not updated the `init_data.py` script so I had to update the script as I had to reinitialize the database because of some alembic issues.

Reply:
Let me pull the new data and try to run this file again.